onvm/onvm_nflib/onvm_event.h

```diff
@@ -126,6 +130,11 @@ struct onvm_event_msg {
         struct rte_mbuf *pkt;
 };
 
+struct onvm_event_data {
+        uint64_t event_id;
+        void *pkt;
+};
```

Comment:
I think we should combine this struct and the `onvm_event_msg` struct -- they seem very similar.  I think your point is that `onvm_event_msg` can only handle a packet, but sometimes we might want different data. I agree with you. Rather than make a new struct I think this could just replace the old one.